charts/zalenium/values.yaml

```diff
@@ -147,8 +147,6 @@ ingress:
   tls: false
   # secretName: my-tls-cert # only needed if tls above is true
   hostname: zalenium.foobar.com
-  # context path without the forward slash (/)
-  # leaving this empty would run zalenium in the root context (/)
```

Comment:
Why are you removing the comments? Perhaps it would be nice to leave something more concrete there explaining what `path` is used for,

Reply:
added appropriate comments.